src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -229,9 +229,7 @@ export default class TableComponent extends ComponentBase implements IComponent,
 			this._table.setSelectedRows(this.selectedRows);
 		}
 
-		for (const col of this._checkboxColumns) {
-			this.registerCheckboxPlugin(col);
-		}
+		Object.keys(this._checkboxColumns).forEach(col => this.registerCheckboxPlugin(this._checkboxColumns[col]));
```

Comment:
@anthonydresser the old code here seems to have stopped working after some lint related cleanup. Please let me know if the new change causes any lint related warning or errors?